multi-agent/lib/mas/elements/providers/mcp_server_client/validator.py

```
@@ -86,10 +85,9 @@ async def _check_connection(
             with anyio.fail_after(context.timeout_seconds):
                 await self._factory.create_async(config, auth_credential=auth_cred)
 
-            elapsed = (time.time() - start) * 1000
             messages.append(self._info(
                 "CONNECTION_OK",
-                f"Connected to MCP server at {config.mcp_url} ({elapsed:.0f}ms)",
+                f"Connected to MCP server",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)